### PR TITLE
[storage] fix comment error about access tier

### PR DESCRIPTION
**Pull request by JasonYang-MSFT**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 
### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).
### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] Validation errors from the [Linter extension for VS Code](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/linter.md) have all been fixed for this spec. (**Note:** for large, previously checked in specs, there will likely be many errors shown. Please contact our team so we can set a timeframe for fixing these errors if your PR is not going to address them).
  - [x] The spec follows the patterns described in the [Swagger good patterns](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-good-patterns.md) document unless the service API makes this impossible.

---

This change is [Reviewable](https://reviewable.io/reviews/azure/azure-rest-api-specs/616)

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@JasonYang-MSFT__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (yabo). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by JasonYang-MSFT** [comment]
https://github.com/Azure/azure-sdk-for-node/issues/1855

File: arm-storage/2016-01-01/swagger/storage.json [hunk 1]
```diff
@@ -463,7 +463,7 @@
       "properties": {
         "name": {
           "type": "string",
-          "description": "Gets or sets the sku name. Required for account creation, optional for update. Note that in older versions, sku name was called accountType.",
+          "description": "Gets or sets the sku name. Required for account creation, optional for update. Note that in older versions, sku name was called accountType. For Blob Storage accounts, only Standard_LRS, Standard_GRS and Standard_RAGRS are supported",
```
Comment (amarzavery) [inline arm-storage/2016-01-01/swagger/storage.json]: Can you remove the phrase Gets or sets? Autorest generated code will automatically have the right prefix based on the property's mutability

Comment (amarzavery) [inline arm-storage/2016-01-01/swagger/storage.json]: > Required for account creation, optional for update.
This statement is weird. 

Does that mean that during update, someone can pass an empty sku object `{}`?
/cc @johanste @anudeepsharma

File: arm-storage/2016-01-01/swagger/storage.json [hunk 2]
```diff
@@ -562,7 +562,7 @@
         },
         "accessTier": {
           "type": "string",
-          "description": "Required for StandardBlob accounts. The access tier used for billing. Access tier cannot be changed more than once every 7 days (168 hours). Access tier cannot be set for StandardLRS, StandardGRS, StandardRAGRS, or PremiumLRS account types.",
+          "description": "Required for StandardBlob accounts. The access tier used for billing. Access tier can only be set for Blob Storage Accounts.",
```
Comment (amarzavery) [inline arm-storage/2016-01-01/swagger/storage.json]: > "Required for StandardBlob accounts. The access tier used for billing. Access tier can only be set for Blob Storage Accounts."
- Which property are you referring to when you talk about "StandardBlob accounts"?
- Which property are you referring to when you talk about "Blob Storage Accounts"?

This is so confusing..

**Review by amarzavery — CHANGES_REQUESTED** [review]

File: arm-storage/2016-01-01/swagger/storage.json (hunk 1 above)
Comment (JasonYang-MSFT) [inline arm-storage/2016-01-01/swagger/storage.json]: Yes

File: arm-storage/2016-01-01/swagger/storage.json (hunk 2 above)
Comment (JasonYang-MSFT) [inline arm-storage/2016-01-01/swagger/storage.json]: Oh, both of them should be Blob Storage Accounts

**Comment by johanste** [comment]
Review status: 0 of 1 files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---

*[arm-storage/2016-01-01/swagger/storage.json, line 466 at r1](https://reviewable.io:443/reviews/azure/azure-rest-api-specs/616#-KU_ZBJBfmDxzVj5ZLlY-r1-466:-KUdM4sKxB89lOZs9rDy:bcu5qdi) ([raw file](https://github.com/azure/azure-rest-api-specs/blob/9e507f57f1ace0da3c028f9ed98b411a3dd7c690/arm-storage/2016-01-01/swagger/storage.json#L466)):*
<details><summary><i>Previously, JasonYang-MSFT (Jason Yang) wrote…</i></summary>
> Yes
</details>
The optional aspect of the comment contradicts the how the model is defined. It should be removed. No tools that look at this swagger (which include autorest, which is used to generated client SDKs) will have the name as optional for update. 

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/azure/azure-rest-api-specs/616)*

**Pull request closed without merging**